Source/Widgets/Animation/Animation.js

```diff
@@ -698,6 +703,31 @@ define([
      * animation.applyThemeChanges();
      */
     Animation.prototype.applyThemeChanges = function() {
+        // Since we rely on computed styles for themeing, we can't actually
+        // do anything if the container has not yet been added to the DOM.
+        // Set up an observer to be notified when it is added and apply
+        // the changes at that time.
+        if (!document.body.contains(this._container)) {
+            if (defined(this._observer)) {
+                //Already listening.
+                return;
+            }
+            var that = this;
+            that._observer = new MutationObserver(function(mutations) {
+                mutations.forEach(function(mutation) {
+                    for (var i = 0; i < mutation.addedNodes.length; i++) {
+                        if (mutation.addedNodes[i].contains(that._container)) {
```

Comment:
Stepping through the code, `contains` works as expected in IE 11 for me.  Are you on the latest version?  Here's what I'm showing: 

![image](https://cloud.githubusercontent.com/assets/1495196/25811458/6188e956-33e1-11e7-92b7-f0dfc73fa3f7.png)